specification/domainservices/resource-manager/Microsoft.AAD/stable/2021-05-01/domainservices.json

```diff
@@ -0,0 +1,1113 @@
+          "description": "A flag to determine whether or not KerberosRc4Encryption is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "KerberosRc4Encryption",
+            "modelAsString": true
+          }
+        },
+        "kerberosArmoring": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Disabled",
+          "description": "A flag to determine whether or not KerberosArmoring is enabled or disabled.",
+          "x-ms-enum": {
+            "name": "KerberosArmoring",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ConfigDiagnostics": {
+      "description": "Configuration Diagnostics",
+      "properties": {
+        "lastExecuted": {
+          "type": "string",
+          "format": "date-time-rfc1123",
+          "description": "Last domain configuration diagnostics DateTime"
+        },
+        "validatorResults": {
+          "type": "array",
+          "description": "List of Configuration Diagnostics validator results.",
+          "items": {
+            "$ref": "#/definitions/ConfigDiagnosticsValidatorResult"
+          }
+        }
+      }
+    },
+    "ConfigDiagnosticsValidatorResult": {
+      "description": "Config Diagnostics validator result data",
+      "properties": {
+        "validatorId": {
+          "type": "string",
+          "description": "Validator identifier"
+        },
+        "replicaSetSubnetDisplayName": {
+          "type": "string",
+          "description": "Replica set location and subnet name"
+        },
+        "status": {
+          "type": "string",
+          "enum": [
+            "None",
+            "Running",
+            "OK",
+            "Failure",
+            "Warning",
+            "Skipped"
+          ],
+          "default": "None",
+          "description": "Status for individual validator after running diagnostics.",
+          "x-ms-enum": {
+            "name": "status",
+            "modelAsString": true
+          }
+        },
+        "issues": {
+          "type": "array",
+          "description": "List of resource config validation issues.",
+          "items": {
+            "$ref": "#/definitions/ConfigDiagnosticsValidatorResultIssue"
+          }
+        }
+      }
+    },
+    "ConfigDiagnosticsValidatorResultIssue": {
+      "description": "Specific issue for a particular config diagnostics validator",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Validation issue identifier."
+        },
+        "descriptionParams": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Single resource property involved in issue description."
+          },
+          "description": "List of domain resource property name or values used to compose a rich description."
+        }
+      }
+    },
+    "CloudError": {
```

Comment:
This whole section is new in the new API version, but it's not part of the diff because the modifications were made before adding the file to the PR. Without the original unchaged file as the baseline, it's not possible to diff within the PR. I had to check out the PR and do the diffs in the file system to discover this addition, which is time-consuming.